release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -510,6 +574,32 @@ warnings.go:67] batch/v1beta1 CronJob is deprecated in v1.22+, unavailable in v1
 
 This is new functionality included with Kubernetes 1.19.
 
+[discrete]
+[id="ocp-4-6-operator-sdk-v-0-19-4"]
+==== Operator SDK v0.19.4
+
+{product-title} supports Operator SDK v0.19.4, which introduces the following
+notable technical changes:
+
+* Operator SDK is now aligned with the {product-title}-wide switch to using UBI-8
+and Python 3. Downstream base images now use UBI-8 and include Python 3.
+* The command `run --local` has been deprecated in favor of `run local`.
+* The commands `run --olm` and `--kubeconfig` have been deprecated in favor of `run packagemanifests`.
+* The default CRD version  has been changed from `apiextensions.k8s.io/v1beta1` to `apiextensions.k8s.io/v1` for commands that create or generate CRDs.
+* The `--kubeconfig` flag has been added to the `<run|cleanup> packagemanifests` command.
+* The `scorecard` command has been removed and replaced by the Technology Preview `alpha scorecard` command, which is an enhanced implementation. See xref:../operators/operator_sdk/osdk-scorecard.adoc#osdk-scorecard[Validating Operators using the scorecard] for more information.
+
+Ansible-based Operator enhancements include:
+
+* The Ansible Operator is now available as a supported release.
+* The Ansible Operator now includes a `healthz` endpoint and `liveness` probe.
+
+Helm-based Operator enhancements include:
+
+* Helm Operators can watch and reconcile when cluster-scoped release resources are changed.
+* Helm Operators can now reconcile logic using three-way strategic merge patches for native Kubernetes objects so that array patch strategies are correctly honored and applied.
```

Comment:
```suggestion
* Helm Operators can now reconcile logic by using three-way strategic merge patches for native Kubernetes objects so that array patch strategies are correctly honored and applied.
```